docs/data/data-grid/row-updates/row-updates.md

```diff
@@ -27,8 +27,10 @@ Alternatively, if you would like to delete a row, you would need to pass an extr
 apiRef.current.updateRows([{ id: 1, _action: 'delete' }]);
 ```
 
-> The free version of the `DataGrid` is limited to a single row update per `apiRef.current.updateRows` call.
-> Multiple row updates at a time are supported in [Pro](/x/introduction/licensing/#pro-plan) and [Premium](/x/introduction/licensing/#premium-plan) plans.
+:::info
+The free version of the `DataGrid` is limited to a single row update per `apiRef.current.updateRows` call.
```

Comment:
IMHO the phrasing is slightly weird
It should either be

> The free version of the Data Grid is limited

Or

> The `DataGrid` component is limited

Because the `DataGrid` format refers to the component which is always free unlike `DataGridPro` and `DataGridPremium`.

If the 2nd proposal is not clear enough it could be something like:

> The `DataGrid` component of the `@mui/x-data-grid` package is limited